scripts/fit_image.py

```diff
@@ -0,0 +1,45 @@
+"""
+A python tool to resize images 
+Usage:
+    python fit_image.py --images xxxx 
+if you wanna set image resolution please use this command :
+    python fit_image.py --images xxxx --width 1920 --height 1080 
+"""
+
+import os
+from PIL import Image
+import argparse
+import re
+
+
+def resize_image(image_path, args):
+    img = Image.open(image_path)
+    img = img.resize([args.width, args.height], Image.BILINEAR)
```

Comment:
We need to keep the ratio when we are resizing an image, automatically.

For example, we have two images, one size is 3000 x 2000, and the other one size is 5000 x 2000, then how do we use your script to resize them?